docs/source/developer_guide/plugins/publish.rst

```diff
@@ -1,41 +1,27 @@
-===================
-Publishing a plugin
-===================
+===========================
+Publishing a plugin package
+===========================
 
 .. _plugins.get_listed:
 
 1. Choose a name
 ----------------
 
-The naming convention for AiiDA plugins is ``aiida-mycode`` for the plugin
-and ``aiida_mycode`` for the corresponding python package, leading to the
-following folder structure::
+The naming convention for AiiDA plugin packages is ``aiida-mycode`` for the plugin distribution on `PyPI`_ and ``aiida_mycode`` for the corresponding python package, leading to the following folder structure::
 
    aiida-mycode/
       aiida_mycode/
          __init__.py
 
-This marks your plugin as an AiiDA package and makes it easy to find on package indices like `PyPI`_.
-
 **Note:** Python packages cannot contain dashes, thus the underscore.
 
 
-2. Get Your Plugin Listed
--------------------------
+2. Register your plugin
+-----------------------
```

Comment:
```suggestion
-------------------------------
```

Reply:
maybe I'm missing something here - just to understand: why make this change?

Reply:
ah, now I saw the comment above. nevermind